addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -1023,8 +1023,9 @@ def _l10n_it_post_invoices_step_2(self, invoices):
         return to_return
 
     def _post_fattura_pa(self, invoice):
-        # OVERRIDE
-        if not invoice.l10n_it_edi_transaction:
+        if not invoice:
```

Comment:
@lordkrandel Is there a case where it can be empty?

Reply:
The method can be called from server actions and traceback

Reply:
Want me to remove it? Let's just merge this.

Reply:
I don't really understand your way of thinking. 
Being defensive against unwanted parameters and conditions (especially when your method in Python is completely public and can be called from whatever code) is something I've been taught all the way from university to 10 years of profession.
I don't know where the confusion could come from: if you pass me no invoice, I'll post no invoice.
But whatever, I've removed it 🤷🏻

Reply:
It depends: https://en.wikipedia.org/wiki/Design_by_contract